shared/styles/css.d.ts

```diff
@@ -4,8 +4,81 @@ import {StyleProp, ViewStyle, TextStyle, ImageStyle} from 'react-native'
 export type Color = null | string
 type _StylesDesktopOverride = {
   wordBreak?: 'normal' | 'break-all' | 'keep-all' | 'inherit' | 'initial' | 'unset' | 'break-word'
+  WebkitAppRegion?: 'drag' | 'no-drag'
 }
-export type _StylesDesktop = CSSProperties & _StylesDesktopOverride
```

Comment:
this style has every conceivable web style, 99% of which we don't use, so in order to speed things up and aid readability we just pull the small subset we use